src/ui/__tests__/components/common/forms/DateInput.test.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import {render} from '@testing-library/react';
+import DateInput from '../../../../components/common/forms/DateInput';
+import '@testing-library/jest-dom';
+import {ChakraProvider} from '@chakra-ui/react';
+import {Formik, Form} from 'formik';
+
+const DEFAULT_PROPS = {
+  name: 'text_field',
+  label: 'Text Field',
+  isRequired: true,
+  helperText: 'Enter text into this field',
+};
+
+describe('TextInput', () => {
+  const setup = (props = DEFAULT_PROPS) => {
+    return render(
+      <ChakraProvider resetCSS>
+        <Formik>
+          {formikProps => (
+            <Form>
+              <DateInput {...props} />
+            </Form>
+          )}
+        </Formik>
+      </ChakraProvider>
+    );
+  };
+
+  it('renders', () => {
+    const component = setup();
+    expect(component).toBeTruthy();
+  });
+
+  it('renders the input field label', () => {
+    const component = setup();
+    const label = component.getByText(DEFAULT_PROPS.label);
+    expect(label).toBeInTheDocument();
+  });
+
+  it('renders the input field with the correct name', () => {
+    const component = setup();
+    const input = component.getByTestId('DateInput');
+    expect(input).toBeInTheDocument();
+    expect(input).toHaveAttribute('name', DEFAULT_PROPS.name);
+  });
+
+  it('renders the helper text', () => {
+    const component = setup();
+    const helperText = component.getByText(DEFAULT_PROPS.helperText);
+    expect(helperText).toBeInTheDocument();
```

Comment:
Nic, sorry for my total ignorance on front end testing.
Does it make sense to have a test that just ensures our Date format is as expexted? i.e. YYYY-MM-DD?

I defer to you on this cus I know nothing about testing front end, I just know the date format is a spot for possible mixups between front-end and backend

Reply:
This is a good question. I had to look up the answer because I didn't know off the top of my head. It appears that HTML inputs with `type="date"` always return values in "YYYY-MM-DD" format.

See: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date

`Note: The displayed date format will differ from the actual value — the displayed date is formatted based on the locale of the user's browser, but the parsed value is always formatted yyyy-mm-dd.`

Because the date is guaranteed to be in that format, I'm going to leave it without testing. It's also difficult (at least for me) to set the value of the input field from inside the test.